frontend/src/components/Header/Presentational.tsx

```diff
@@ -21,17 +20,7 @@ export const Presentational: React.FunctionComponent<IProps> = (props) => {
   return (
     // TODO: make header show userInformation.fullname when backend supports it.
     <div id="student-header">
-      You're logged in as {userInformation.username}
-      <Link to="/admin">
-        <KnappBase
-          type="hoved"
-          htmlType="button"
-          id="logoutButton"
-          onClick={onClickAction}
-        >
-          Admin
-        </KnappBase>
-      </Link>
+      You're logged in as {userInformation.fullname}
```

Comment:
Why is it red?

Reply:
> Why is it red?

I think it' s just a bug on Github, it happens sometimes.

Reply:
The red background-color is definitely being caused by the error-highlighting feature of GitHub's text editor/viewer. You can find the same behaviour occurring to an intended block comment in another JSON file on GitHub:

Reply:
I think the correct syntax would be to wrap the entire thing in a formatted string, like:
`{'You're logged in as ${userinformation.fullname}'`

Reply:
```suggestion
      {`You're logged in as ${userInformation.fullname}`}
```